contracts/ShortNameClaims.sol

```diff
@@ -52,30 +54,60 @@ contract ShortNameClaims is Ownable {
         Status status;
     }
 
+    Roles.Role owners;
+    Roles.Role ratifiers;
+
     PriceOracle public priceOracle;
     BaseRegistrar public registrar;
     mapping(bytes32=>Claim) public claims;
     mapping(bytes32=>bool) approvedNames;
     uint public pendingClaims;
     uint public unresolvedClaims;
     Phase public phase;
-    address public ratifier;
 
     event ClaimSubmitted(string claimed, bytes dnsname, uint paid, address claimant, string email);
     event ClaimStatusChanged(bytes32 indexed claimId, Status status);
 
     constructor(PriceOracle _priceOracle, BaseRegistrar _registrar, address _ratifier) public {
         priceOracle = _priceOracle;
         registrar = _registrar;
-        ratifier = _ratifier;
         phase = Phase.OPEN;
+
+        owners.add(msg.sender);
+        ratifiers.add(_ratifier);
+    }
+
+    modifier onlyOwner() {
+        require(owners.has(msg.sender), "Caller must be an owner");
+        _;
+    }
+
+    modifier onlyRatifier() {
+        require(ratifiers.has(msg.sender), "Caller must be a ratifier");
+        _;
     }
 
     modifier inPhase(Phase p) {
         require(phase == p, "Not in required phase");
         _;
     }
 
+    function addOwner(address owner) external onlyOwner {
+        owners.add(owner);
+    }
+
+    function removeOwner(address owner) external onlyOwner {
+        owners.remove(owner);
+    }
+
+    function addRatifier(address ratifier) external onlyRatifier {
+        ratifiers.add(ratifier);
```

Comment:
Do you know who to add as an ratifier when deploying the contract? If initial ratifier is empty, you cannot add additional ratifies. Alternatively you can let owner to add ratifier, too.

Reply:
I'll add my own account, since we're just using it for coordination.

If owners could add ratifiers, then any owner could do ratification themselves, and there would be no point in having separate roles.